openfpga/src/base/openfpga_setup_command_template.h

```diff
@@ -1181,6 +1226,13 @@ void add_setup_command_templates(openfpga::Shell<T>& shell,
   add_write_fabric_pin_physical_location_command_template<T>(
     shell, openfpga_setup_cmd_class,
     cmd_dependency_write_fabric_pin_physical_location, hidden);
+
+  /********************************
+   * Command 'report_reference'
+   */
+  std::vector<ShellCommandId> cmd_dependency_report_reference;
+  add_report_reference_command_template<T>(
```

Comment:
Add dependency for the command. It should be called after the command ``build_fabric`` is called.